src/hyp3_isce2/s1_auxcal.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021-present Caltech
+# Modifications Copyright 2023 Alaska Satellite Facility
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import zipfile
+from io import BytesIO
+from pathlib import Path
+from typing import Union
+
+import requests
+
+
+S1A_AUX_URL = 'https://sar-mpc.eu/download/55282da1-679d-4ecf-aeef-d06b024451cf'
+S1B_AUX_URL = 'https://sar-mpc.eu/download/3c8b7c8d-d3de-4381-a19d-7611fb8734b9'
+
+
+def _download_platform(url: str, aux_cal_dir: Path):
```

Comment:
In general I like the use of leading `_` for internal module components, but are we using that convention consistently across this repo? If not, we should probably make that decision.

Reply:
I think we should commit to always using the leading `_` for non-public functions.

Reply:
Sounds good, maybe we should make that change to all our non-public components before merging to `main`.

Reply:
I created an issue: https://github.com/ASFHyP3/hyp3-isce2/issues/13